test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -1186,4 +1191,51 @@ def test_non_truncate_id_for_non_amex_transactions
     assert_equal 31, response.params['reference'].length
     assert_equal 'Visa', response.params['source']['scheme']
   end
+
+  def test_successful_purchase_with_minimun_lvl_2_data
+    response = @gateway.purchase(@amount, @credit_card, @minimun_lvl2_options)
+    assert_success response
+  end
+
+  def test_successful_purchase_with_minimun_lvl_2_data_for_amex
+    amex_options = {
+      currency: 'USD',
+      line_items: [{ item_name: 'Paint', quantity: '1', unit_cost: '1270' }]
+    }
+    @amex_card = credit_card('345678901234564', brand: 'american_express', verification_value: '1000', month: '12', year: Time.now.year)
+
+    response = @gateway.purchase(1500, @amex_card, @minimun_lvl2_options.merge(amex_options))
+    assert_success response
+  end
+
+  def test_successful_purchase_with_minimun_lvl_3_data
```

Comment:
if is possible, would be good idea add a capture request with l2/l3 data as well.